### PR TITLE
docs: remove `display: block;` from `<main>`

### PR DESCRIPTION
**Pull request by adamjohnson**
A few elements had `main { display: block; }` left over from when the `<main>` tag in the demos was set to `display: grid;`—which caused [some layout issues](https://redhat-internal.slack.com/archives/C05GTN4E82G/p1722367349722359) in certain demos.

## What I did

1. I removed the now-defunct `display: block;` declarations from various demo files.

## Testing Instructions

View the standalone / full page DP demos of the following components and make sure they look normal:

  * `rh-breadcrumb`
  * `rh-card` (promo only)
  * `rh-tile`
  * `rh-video-embed`

### CODE REVIEW

**Review by markcaron — APPROVED** [review]
Layout Goof? Trim `<Main>`

**Comment by adamjohnson** [comment]
> @adamjohnson looks like there's a few linting issues remaining otherwise.

Yup, AFAICT those are upstream. Might just sit on those until they are resolved upstream then merge this. No rush.

File: elements/rh-badge/rh-badge.css
Comment (bennypowers) [inline elements/rh-badge/rh-badge.css:1]: this would change rh-badge, perhaps this file was included by mistake?

**Review by bennypowers — CHANGES_REQUESTED** [review]
other than rh-badge.css, lgtm

**Review by bennypowers — APPROVED** [review]

**Pull request merged**